pkgs/development/compilers/purescript/purescript/test-minimal-module/default.nix

```diff
@@ -3,9 +3,9 @@
 runCommand "purescript-test-minimal-module" {} ''
   ${purescript}/bin/purs compile -o ./output ${./.}/Main.purs
 
-  echo 'require("./output/Main/index.js").main()' > node.js
+  echo 'import {main} from "./output/Main/index.js"; main()' > node.mjs
 
-  ${nodejs}/bin/node node.js | grep "hello world" || echo "did not output hello world"
+  ${nodejs}/bin/node node.mjs | grep "hello world" || (echo "did not output hello world"; exit 1)
```

Comment:
Just for posterity, this was needed because of the switch from Common JS to ES modules.